fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -224,6 +224,9 @@ public enum OlapTableState {
     @SerializedName(value = "maxColUniqueId")
     protected int maxColUniqueId = -1;
 
+    @SerializedName(value = "maxIndexId")
+    protected long maxIndexId = -1;
```

Comment:
we can make this new feature serialized after all changes are merged.

Reply:
I think it is acceptable because this feature is under implementing no test will depend on it.
And -1 is not a good choice for the default number because -1 will be the index number for existing indexes if I'm not mistaken.